cspell.json

```diff
@@ -4,44 +4,36 @@
   "dictionaryDefinitions": [],
   "dictionaries": [],
   "words": [
-    // replugged terminology
```

Comment:
Any reason why you removes these?

Reply:
I didn't do that myself; I think either the Prettier or eslint plugin in VSCode did that automatically when I edited that file, because it sees comments in JSON and labels them (correctly) as invalid.

Reply:
Interesting